torch/csrc/jit/script/compiler.cpp

```diff
@@ -2020,6 +2015,21 @@ struct to_ir {
       return NamedValue(attr.range(), attr.name().name(), emitExpr(attr.value()));
     });
   }
+
```

Comment:
Weird function, maybe instead:

```
void checkSpecialApply(const Apply& apply, size_t expected_inputs)
```

Current name suggests it is used for all apply expressions, and is weirdly hardcoded for the case where there are 2 inputs.